Source/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -186,16 +186,11 @@ - (void)setAlpha:(CGFloat)newAlpha
 - (CGFloat)cornerRadius
 {
   ASDN::MutexLocker l(__instanceLock__);
-  if (_cornerRoundingType == ASCornerRoundingTypeDefaultSlowCALayer) {
-    return self.layerCornerRadius;
-  } else {
-    return _cornerRadius;
-  }
+  return _cornerRadius;
 }
 
 - (void)setCornerRadius:(CGFloat)newCornerRadius
 {
-  ASDN::MutexLocker l(__instanceLock__);
```

Comment:
`-updateCornerRoundingWithType:cornerRadius:` acquires the lock itself ([here](https://github.com/TextureGroup/Texture/pull/465/files#diff-fb6dadff65468c53595c164828199c83R1549)).